build/common.go

```diff
@@ -76,24 +76,35 @@ func buildBackend(cfg Config) error {
 		return err
 	}
 
-	// Link statically
-	ldFlags := `-extldflags "-static"`
-	if !cfg.EnableDebug {
-		// Add linker flags to drop debug information
-		ldFlags = "-w -s " + ldFlags
-	}
-
 	args := []string{
-		"build", "-o", path.Join("dist", exeName), "-ldflags", ldFlags,
+		"build", "-o", path.Join("dist", exeName),
+	}
+	if !cfg.EnableDynamicLinking {
+		// Link statically
+		ldFlags := `-extldflags "-static"`
+		if !cfg.EnableDebug {
+			// Add linker flags to drop debug information
+			ldFlags = "-w -s " + ldFlags
+		}
+		args = append(args, "-ldflags", ldFlags)
+	} else {
+		args = append(args, "-tags", "netgo")
```

Comment:
`-tags netgo` shouldn't be relevant any longer, as you can see [here](https://golang.org/doc/go1.5#net) it's controlled at runtime. You can also see [here](https://golang.org/pkg/net/#hdr-Name_Resolution) that the Go resolver is used by default.
```suggestion
```

Reply:
so in case of dynamic linking no other arguments are required?

Reply:
Yeah, basically as I describe in other comments, I realized in the meantime that it should be about enabling cgo or not. Static linking is just an effect of disabling cgo.